demos-europe/demosplan-core
# Fix(refs:35060): remove required symbol for emailAddress

**Ticket:** https://yaits.demos-deutschland.de/T35060

### Description: 

- the email field should not have the mandatory symbol, because its not mandatory here
- the phoneNumber field shall be reqquired

### How to review/test
- login as mandant admin and look how the label for email field looks like when you wanna create a new support contact
- try to save a contact without phoneNumber. you should get an error and it should fail


### PR Checklist
